src/Adapter/Product/Image/CommandHandler/SetProductImagesForAllShopHandler.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Product\Image\CommandHandler;
+
+use Image;
+use PrestaShop\PrestaShop\Adapter\Product\Image\Repository\ProductImageMultiShopRepository;
+use PrestaShop\PrestaShop\Adapter\Product\Image\Repository\ProductImageRepository;
+use PrestaShop\PrestaShop\Adapter\Product\Repository\ProductMultiShopRepository;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\Command\ProductImageSetting;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\Command\SetProductImagesForAllShopCommand;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\CommandHandler\SetProductImagesForAllShopHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\Exception\CannotRemoveCoverException;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\ValueObject\ImageId;
+use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopId;
+
+class SetProductImagesForAllShopHandler implements SetProductImagesForAllShopHandlerInterface
+{
+    /**
+     * @var ProductImageRepository
+     */
+    private $productImageRepository;
+
+    /**
+     * @var ProductImageMultiShopRepository
+     */
+    private $productImageMultiShopRepository;
+    /**
+     * @var ProductMultiShopRepository
+     */
+    private $productMultiShopRepository;
+
+    public function __construct(
```

Comment:
I see we don't require the phpdocs anymore if props are all typed. Im ok with that :+1:

Reply:
I don't think it worth it? @jolelievre what about this?